website/docs/tutorials/07-testing/02-cheatcodes/reflect.md

```diff
@@ -0,0 +1,65 @@
+# `reflect`
+```python
+def reflect(self, ids: VmConsts) -> Reflector:
+Reflector.get() -> Union[CairoStruct, RelocatableValue, int]
+def CairoStruct(typename: str) -> Type # inherits from base CairoStruct
+```
+Lazily converts Cairo object into Python custom named `CairoStruct` (complex structure) or keeps it a simple type `RelocatableValue` (pointer) or `int` (felt). It can be used to easily print and compare complex structures.
+```cairo title="./test/example_test.cairo"
+%lang starknet
```

Comment:
I feel this example is a bit too complex for docs. Can we provide one simple example and then a complex one? Also please change variable names to more descriptive names than `a`, `b`, `c`